Upstatement/skela-wp-theme
# Remove jquery as a dependency

## Changes

Removes jQuery

## How To Test

- `npm install`
- `./bin/start` and make sure the 💀 emoji shows up in the console
